Library/Formula/makensis.rb

```diff
@@ -30,10 +43,20 @@ def install
     ENV.libstdcxx if ENV.compiler == :clang
 
     # Don't strip, see https://github.com/Homebrew/homebrew/issues/28718
-    scons "STRIP=0", "makensis"
-    bin.install "build/release/makensis/makensis"
+    scons "STRIP=0", "SKIPUTILS=all", "makensis"
```

Comment:
The flag will skip building Win32 tools that are part of NSIS but not required to compile scripts on the platform. This includes *NSIS Menu*, an application that offers nothing but links to NSIS resources on the computer and online. Another utility is `zip2exe`, which converts archives to simple installers. So, both are convenience tools that are of no requirement by the makensis compiler.

The [building documentation](http://nsis.sourceforge.net/Docs/AppendixG.html#build_posix) actually suggests a set of additional flags for POSIX platforms, but as far as I can tell we need the stubs and plugins. I only came across the `SKIPUTILS` flag, since NSIS 3.0 would not build without `zlib (win32)`, and I wouldn't know how to include that into the formula. Apart from that, NSIS developer @kichik mentioned that to his knowledge zlib is only required for the `zip2exe` utility.

Hope this helps!